### PR TITLE
Experius 2.1 patch empty country options in checkout

### PR DESCRIPTION
**Pull request by tools-utrecht-hh**
In the checkout, the country select contains empty values. This commit will fix this issue.

### Description
In the country select option in the checkout there are empty values.

### Manual testing scenarios

1. Enable country in checkout
2. Have at least 2 or more available countries configured
3. Add a product to your cart
4. Go to the checkout
5. Open the country select
country

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x ] All commits are accompanied by meaningful commit messages
 - [x ] All new or changed code is covered with unit/integration tests (if applicable)
 - [x ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=13363) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/magento2?pullRequest=13363) before we can accept your contribution.<br/><hr/>**Maikel Bakker** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

**Comment by miguelbalparda** [comment]
@experius-nl keep in mind you will need to sign the CLA with the same user who did the commits.

File: app/code/Magento/Directory/Model/ResourceModel/Country/Collection.php [hunk 1]
```diff
@@ -237,7 +237,7 @@ public function addCountryIdFilter($countryId)
      * @param string|boolean $emptyLabel
      * @return array
      */
-    public function toOptionArray($emptyLabel = ' ')
+    public function toOptionArray($emptyLabel = false)
```
Comment (sidolov) [inline app/code/Magento/Directory/Model/ResourceModel/Country/Collection.php:240]: Provided changes are backward incompatible, please, follow [Backward compatible development documentation](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/#modifying-the-default-values-of-optional-arguments-in-public-and-protected-methods)

**Review by sidolov — CHANGES_REQUESTED** [review]
Please, sign CLA, otherwise we can't process your PR

**Comment by okorshenko** [comment]
Hi @experius-nl 
Thank you for your contribution. We cann't accept such changes due to backward compatibility policy

**Pull request closed without merging**